temporal-sdk/src/test/java/io/temporal/workflow/DescribeTest.java

```diff
@@ -175,12 +175,15 @@ public void testSuccessfulActivity() throws InterruptedException {
             .toBuilder()
             .setHeartbeatDetails(DescribeWorkflowAsserter.stringsToPayloads("heartbeatDetails"))
             .setLastHeartbeatTime(actual.getLastHeartbeatTime())
-            .setLastWorkerIdentity(
-                testWorkflowRule
-                    .getTestEnvironment()
-                    .getWorkflowClient()
-                    .getOptions()
-                    .getIdentity())
+            // TODO: Uncomment when the server is fixed. We should always expect to see an identity
```

Comment:
But it needs to be higher then? On line 140 where the expected is created for the first check. Because this field should be published even before the heartbeat.
May be worth relocating if you want to keep it in a commented way.